miso-web/src/main/webapp/scripts/list_servicerecord.js

```diff
@@ -113,8 +116,19 @@ ListTarget.servicerecord = {
             });
             list += "</ul>";
             return list;
+          } else {
+            var filenames = "";
+            var counts = 0;
+            full.attachments.forEach(function (file) {
+              if (full.attachments.length - 1 == counts) {
+                filenames += file.filename;
+              } else {
+                filenames += file.filename + " ";
+                ++counts;
+              }
+            });
+            return filenames;
```

Comment:
@callunity made a suggestion to shorten this significantly using `map` and `join`